legal/association_statute.md

```diff
@@ -0,0 +1,224 @@
+Rodział XI: Sposób reprezentacji
+
+§49 Do składania oświadczeń woli w imieniu Stowarzyszenia, w tym w sprawach majątkowych, uprawnionych jest dwóch członków Zarządu działających łącznie.
+
+Rozdział XII: Działalność gospodarcza
+
+§50 Stowarzyszenie może prowadzić działalność gospodarczą na ogólnych zasadach określonych w odrębnych przepisach. Stowarzyszenie prowadzi działalność gospodarczą wyłącznie w rozmiarach służących realizacji celów statutowych.
+
+§51 Stowarzyszenie prowadzi działalność gospodarczą w następujących dziedzinach:
+a) Sprzedaż detaliczna prowadzona przez domy sprzedaży wysyłkowej lub Internet - 47.91.Z,
+b) Wydawanie książek - 58.11.Z,
+c) Wydawanie wykazów oraz list (np. adresowych, telefonicznych) - 58.12.Z,
+d) Wydawanie czasopism i pozostałych periodyków - 58.14.Z,
+e) Pozostała działalność wydawnicza - 58.19.Z,
+f) Działalność wydawnicza w zakresie gier komputerowych - 58.21.Z,
+g) Działalność wydawnicza w zakresie pozostałego oprogramowania - 58.29.Z
+h) Działalność związana z produkcją filmów, nagrań wideo i programów telewizyjnych - 59.11.Z,
+i) Działalność postprodukcyjna związana z filmami, nagraniami wideo i programami telewizyjnymi - 59.12.Z,
+j) Działalność związana z dystrybucją filmów, nagrań wideo i programów telewizyjnych - 59.13.Z,
+k) Przetwarzanie danych; zarządzanie stronami internetowymi (hosting) i podobna działalność - 63.11.Z,
+l) Działalność portali internetowych - 63.12.Z,
+m) Pozostała działalność usługowa w zakresie informacji, gdzie indziej niesklasyfikowana - 63.99.Z,
+n) Działalność związana z oprogramowaniem - 62.01.Z,
+o) Działalność związana z doradztwem w zakresie informatyki - 62.02.Z,
+p) Działalność związana z zarządzaniem urządzeniami informatycznymi - 62.03.Z,
+q) Pozostała działalność usługowa w zakresie technologii informatycznych i komputerowych - 62.09.Z,
+r) Badania naukowe i prace rozwojowe w dziedzinie pozostałych nauk przyrodniczych i technicznych - 72.19.Z,
+s) Badania naukowe i prace rozwojowe w dziedzinie nauk społecznych i humanistycznych - 72.20.Z,
+t) Działalność agencji reklamowych - 73.11.Z,
+u) Działalność związana z organizacją targów, wystaw i kongresów - 82.30.Z,
+v) Pozostałe pozaszkolne formy edukacji, gdzie indziej niesklasyfikowane - 85.59.B,
+w) Działalność pozostałych organizacji członkowskich, gdzie indziej niesklasyfikowana - 94.99.Z.
+
+Rozdział XIII: Postanowienia końcowe
+
+§52 Uchwałę w sprawie zmiany statutu Stowarzyszenia podejmuje Walne Zebranie Członków zwykłą większością głosów, w obecności przynajmniej połowy członków uprawnionych do głosowania.
+
+§53 Uchwałę o rozwiązaniu Stowarzyszenia podejmuje Walne Zebranie Członków zwykłą większością głosów przy obecności co najmniej połowy członków uprawnionych do głosowania.
+
+§54 Podejmując uchwałę o rozwiązaniu Stowarzyszenia, Walne Zebranie Członków określa jego sposób likwidacji oraz przeznaczenia majątku Stowarzyszenia.
+
+
+Statut przyjęto uchwałą w dniu ..........
+
+Podpisane:
+.......
+.......
```

Comment:
Czy statut stowarzyszenia musi być podpisany? Jeśli tak to lepiej skasować wykropkowane miejsca